deploy/src/bin/thing-flinger.rs

```diff
@@ -501,7 +673,11 @@ fn overlay_rss_config(
 ) -> Result<()> {
     // Sync `config-rss.toml` to the directory for the RSS server on the
     // builder.
-    let src = config.omicron_path.join("smf/sled-agent/config-rss.toml");
+    let src = if let Some(src) = &config.rss_config_path {
+        src.clone()
+    } else {
+        config.omicron_path.join("smf/sled-agent/config-rss.toml")
+    };
     let dst = format!(
         "{}@{}:{}",
```

Comment:
I think this should be:

```suggestion
        "{}@{}:{}/config-rss.toml",
```

Otherwise, if `config.rss_config_path` points to a file whose basename isn't exactly `config-rss.toml`, it will copy the file as named (which will be ignored by RSS).

Reply:
Good catch! That is more flexible. Thanks!